### PR TITLE
Change the linter to flutter_lints

### LINKED ISSUE #24: Migrate from package:pedantic to package:lints
**Describe the solution you'd like**

Per the guidance from [dart-lang/lints](https://github.com/dart-lang/lints#migrating-from-packagepedantic):

in `pubspec.yaml`:

```diff
-  pedantic: ^1.11.0
+  lints: ^1.0.1
```

and in each `analysis_options.yaml`:

```diff
-include: package:pedantic/analysis_options.yaml
+include: package:lints/recommended.yaml
```

### PR DESCRIPTION
**Pull request by prasadsunny1**
Fixes #24 

**- What I did**
Change the linter package from pedantic to flutter_lints

**- How I did it**
Change the dependency

**- How to verify it**
Inspect the file `pubspec.yaml`

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -184,7 +198,7 @@ packages:
     source: hosted
     version: "1.8.0"
   pedantic:
```
Comment (cpswan) [inline pubspec.lock]: I'm not sure there should even be a pedantic section here any more, which is why I've asked @XavierChanth to review.

There may also be a need for an analysis-options.yaml

Comment (prasadsunny1) [reply pubspec.lock]: Will add that

Comment (prasadsunny1) [reply pubspec.lock]: Done, I have added analysis options and some refactorings that it suggested.

**Comment by prasadsunny1** [comment]
Added analysis_option.yaml
Resolved some items suggested by flutter_lints
Refreshed pubspec.lock file

File: lib/src/command_runner.dart [hunk 2]
```diff
@@ -41,7 +43,7 @@ class AtCommandRunner extends CommandRunner<CommandStatus> {
   @override
   Future<CommandStatus?> runCommand(ArgResults topLevelResults) async {
     if (topLevelResults['version']) {
-      print('at_app ${packageVersion.toString()}');
+      log('at_app ${packageVersion.toString()}');
```
Comment (xavierchanth) [inline lib/src/command_runner.dart]: Good catch, this should use the existing logger in the class though:
```
_logger.i('...');
```

File: analysis_options.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# This file configures the analyzer, which statically analyzes Dart code to
+# check for errors, warnings, and lints.
+#
+# The issues identified by the analyzer are surfaced in the UI of Dart-enabled
+# IDEs (https://dart.dev/tools#ides-and-editors). The analyzer can also be
+# invoked from the command line by running `flutter analyze`.
+
+# The following line activates a set of recommended lints for Flutter apps,
+# packages, and plugins designed to encourage good coding practices.
+include: package:flutter_lints/flutter.yaml
```
Comment (xavierchanth) [inline analysis_options.yaml:1]: This is a dart project, it should use the [dart lint options](https://pub.dev/packages/lints).

File: lib/src/util/cli/flutter.dart [hunk 4]
```diff
@@ -41,8 +41,9 @@ class Flutter {
     if (org != null) args.add('--org=$org');
     if (projectName != null) args.add('--project-name=$projectName');
     if (iosLanguage != null) args.add('--ios-language=$iosLanguage');
-    if (androidLanguage != null)
+    if (androidLanguage != null) {
```
Comment (xavierchanth) [inline lib/src/util/cli/flutter.dart]: I used shorthand in these lines for readability, please try to keep the formatting the same.

**Review by xavierchanth — CHANGES_REQUESTED** [review]
Thank you for you PR, there are a few changes I am requesting:

- The log statements should use the custom logger
- No flutter in this project so please use package:lints instead

**Pull request merged**